autogtp/Management.cpp

```diff
@@ -383,6 +383,26 @@ void Management::fetchNetwork(const QString &name) {
     return;
 }
 
+
+void Management::archiveFiles(const QString &sgf_file, const QString &data_file, const QString &debug_file) {
+    QDir dir;
+    if (!m_keepPath.isEmpty()) {
+        QFile(sgf_file).copy(m_keepPath + '/' + sgf_file);
+    }
+    dir.remove(sgf_file);
+
+    if (!m_debugPath.isEmpty()) {
+        if(!data_file.isEmpty()) {
```

Comment:
@marcocalignano Are you sure if all the training data (data_file) correctly deleted when --debug is not present?

Reply:
Good catch. Thanks!